### PR TITLE
ISSUE-1220 Enable RabbitMQAndRedisEventBusWithRedisTLSSentinelTest

### PR DESCRIPTION
**Pull request by hungphan227**

### CODE REVIEW

File: tmail-backend/pom.xml [hunk 1]
```diff
@@ -27,15 +27,15 @@
...
+<!--        <module>deployment-tests</module>-->
 
         <module>guice/blob-guice</module>
         <module>guice/distributed</module>
         <module>guice/encrypted-mailbox</module>
         <module>guice/protocols/jmap</module>
         <module>guice/team-mailboxes</module>
 
-        <module>integration-tests</module>
+<!--        <module>integration-tests</module>-->
```
Comment (quantranhong1999) [inline tmail-backend/pom.xml:38]: Why this change?

Comment (hungphan227) [reply tmail-backend/pom.xml:38]: Just to make CI run faster

Comment (hungphan227) [reply tmail-backend/pom.xml:38]: no merge

**Comment by quantranhong1999** [comment]
Can rebase.
